src/main/java/uk/gov/hmcts/reform/idam/client/IdamApi.java

```diff
@@ -45,6 +45,14 @@ Response authenticatePinUser(
         @RequestParam("state") final String state
     );
 
+
+    /**
+     * User Authenticate method.
+     *
+     * @deprecated
+     * IDAM oauth2/authorize endpoint is deprecated
```

Comment:
what's the method to be used if this is deprecated :)

Reply:
end point deprecated. no needed.